docs/dev/upgrade-opentelemetry.md

```diff
@@ -4,18 +4,22 @@ When the [opentelemetry-collector](https://github.com/open-telemetry/opentelemet
 
 ## Update All Dependencies
 
-There is a `make` target that will update all `opentelemetry-collector-contrib` dependencies, which will include all indirect `opentelemetry-collector` dependencies:
+There is a `make` target to update all OpenTelemetry dependencies and regenerate necessary metadata. To run the update:
+```
+make update-opentelemetry
+```
+
+Read on for details of the actual details of the target, or skip ahead to [Test](#test) to continue the process.
```

Comment:
[Optional] Want to make it a collapsible zippy section (see, e.g., https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)?

Reply:
Great idea, this is done